packages/@aws-cdk/aws-lambda-nodejs/README.md

```diff
@@ -43,6 +43,7 @@ All other properties of `lambda.Function` are supported, see also the [AWS Lambd
 
 The `NodejsFunction` construct automatically [reuses existing connections](https://docs.aws.amazon.com/sdk-for-javascript/v2/developer-guide/node-reusing-connections.html)
 when working with the AWS SDK for JavaScript. Set the `awsSdkConnectionReuse` prop to `false` to disable it.
+This must be disabled when the function is meant to be used as a lambda@edge.
```

Comment:
Is it possible to identify that this lambda is used as Lambda@Edge and make the default smarter?

@jogold, any ideas?

Reply:
Yes this is #9562